scripts/build/architecture-convert.sh

```diff
@@ -51,17 +51,25 @@ if [ "$ARGV_TYPE" == "node" ]; then
     RESULT=ia32
   elif [ "$ARGV_ARCHITECTURE" == "x64" ]; then
     RESULT=x64
-  elif [ "$ARGV_ARCHITECTURE" == "armv7l" ]; then
+  elif [ "$ARGV_ARCHITECTURE" == "armhf" ]; then
```

Comment:
I guess it's just a "belt and braces" approach, but you're right in that it's probably not necessary. As long as we're always sure that the type-blocks are always checking for the identical ARCH strings ;)